load-balancer/README.md

```diff
@@ -0,0 +1,14 @@
+# Load balancer
+This recipe shows a Service Worker containing network logic to dynamically select the best content provider accordingly to server availability.
+
+## Difficulty
+Intermediate
+
+## Use case
+As a service provider, I want to provide the best source in terms of availability for a selected resource.
+
+## Solution
+Use a Service Worker to intercept the requests to the resources and select the proper content provider accordingly to their availability.
+
+## Category
+More than offline
```

Comment:
Done, if you want to take a deeper look, here we are a little bit busy and it's the last recipe I need to publish the blog post. Thx.